lib/dradis/plugins/projects/upload/v1/template.rb

```diff
@@ -347,7 +358,26 @@ def parse_report_content(template)
           content_library.save
         end
 
-        logger.info { 'Done.' }
+        logger.info { 'Done processing document properties.' }
+      end
+
+      def parse_content_blocks(template)
+        return unless defined?(ContentBlock)
+
+        logger.info { 'Processing content blocks...' }
+
+        template.xpath('//content_blocks/content_block').each do |xml_block|
+          block_attributes = {
+            author: xml_block.at_xpath('author').text.strip,
+            kind: xml_block.at_xpath('kind').text,
+            text: xml_block.at_xpath('text').text,
```

Comment:
new changes that will impact this:
`author` should be a `User` object
`kind` renamed to `name`
`text` renamed to `content`